src/main/docs/guide/securityRule.adoc

```diff
@@ -1,28 +1,44 @@
 The decision to allow access to a particular endpoint to anonymous or authenticated users is determined by a collection of
-Security Rules. Micronaut ships with several built-in security rules. If they don't fulfil your needs,
+Security Rules which are executed from the api:security.filter.SecurityFilter[]. Micronaut ships with several built-in security rules. If they don't fulfil your needs,
 you can implement your own api:security.rules.SecurityRule[].
 
-Security rules implement the ordered interface and they are executed in order. All of the existing rules have a static variable `ORDER` that stores the order of that rule. You can use those variables to place your own rule before or after any of the existing rules.
-
 Security rules return a publisher that should emit a single api:security.rules.SecurityRuleResult[]. See the following table for a description of each result.
 
 |===
 |Result |Description
 
 |api:security.rules.SecurityRuleResult#ALLOWED[ALLOWED]
-|Access to the resource should be granted and no further rules will be considered.
+|Access to the resource should be granted and *no further rules will be considered*.
 
 |api:security.rules.SecurityRuleResult#REJECTED[REJECTED]
-|Access to the resource should be rejected and no further rules will be considered.
+|Access to the resource should be rejected and *no further rules will be considered*.
 
 |api:security.rules.SecurityRuleResult#UNKNOWN[UNKNOWN]
 |The rule doesn't apply to the request resource, or it cannot be determined either way. This result will cause other security rules to be considered.
 |===
 
 IMPORTANT: If all security rules return `UNKNOWN`, the request will be rejected!
 
+IMPORTANT: This implementation of api:security.filter.SecurityFilter[] so the security rules are exclusive: The first that manages the request (which returns alternatively `ALLOWED` or `REJECTED`) will be *the only one considered*.
```

Comment:
```suggestion
IMPORTANT: api:security.filter.SecurityFilter[] evaluates security rules in order. **The remaining rules are not evaluated once a rule returns `ALLOWED` or `REJECTED`**.
```

Reply:
What I wrote here actually didn't have any sense. I guess I've got tricked by copy and paste 😅